frontend/src/components/TournamentTeam.js

```diff
@@ -140,12 +162,17 @@ const TournamentTeam = () => {
       </h2>
 
       <For each={Object.entries(matchesGroupedByDate())}>
-        {/* eslint-disable-next-line no-unused-vars */}
-        {([tournamentDate, matches], idx) => (
+        {([tournamentDate, matches]) => (
           <div class="mb-10">
-            <div class="mb-5 ml-1">
-              <h3 class="text-center text-lg font-bold">Day - {idx() + 1}</h3>
-            </div>
+            <Show when={tournamentDates().length > 1}>
```

Comment:
Because of the below grouping, day 1 will not be present in the grouped matches, only day 2 will be there right ?
```js
     const teamMatches = matchesQuery.data?.filter(
        match =>
          match.team_1?.ultimate_central_slug === params.team_slug ||
          match.team_2?.ultimate_central_slug === params.team_slug
      );
     
      setMatchesGroupedByDate(
        Object.groupBy(teamMatches, ({ time }) =>
          new Date(Date.parse(time)).getUTCDate()
        )
      );
```

Reply:
> Because of the grouping, day 1 will not be present in the grouped matches, only day 2 will be there right ?
> 
> ```js
>      const teamMatches = matchesQuery.data?.filter(
>         match =>
>           match.team_1?.ultimate_central_slug === params.team_slug ||
>           match.team_2?.ultimate_central_slug === params.team_slug
>       );
>      
>       setMatchesGroupedByDate(
>         Object.groupBy(teamMatches, ({ time }) =>
>           new Date(Date.parse(time)).getUTCDate()
>         )
>       );
> ```

Reply:
Oh cool perfect